packages/neuron-ui/src/styles/index.scss

```diff
@@ -269,3 +269,16 @@ body {
     border-radius: 48px;
   }
 }
+
+.infinite-rotation {
```

Comment:
For cases where class selectors need to be defined, I suggest using `css module` as much as possible.
Consider creating a new `styles/index.module.scss` or `styles/animation.module.scss` file.